### PR TITLE
Release 2.0.3

### PR DESCRIPTION
**Pull request by stephen-cox**
Changelog:

#83 @ekes @andybroomfield Update composer dependencies to non-alpha localgov modules
#105 @ekes @Adnan-cds Remove rather than hide search
#139 @stephen-cox @finnlewis Update blocks to use the new LocalGov Base theme
#140 @stephen-cox @finnlewis Require localgov_core ^2.1

### CODE REVIEW

**Comment by ekes** [comment]
#82 was merged into 1.x so changelog is #105 which is the bug fix I'd worked on; #139 and #140 are straightforward version compatibility updates.

Changelog:

#105 @ekes @Adnan-cds Remove rather than hide search
#139 @stephen-cox @finnlewis Update blocks to use the new LocalGov Base theme
#140 @stephen-cox @finnlewis Require localgov_core ^2.1

File: README.md [hunk 1]
```diff
@@ -69,4 +89,15 @@ and it will be exported with other configuration. Any types that exist in
 configuration will be imported.
 
 ## Block placement
-When using a theme other than the localgov_theme, the **Directory channel search** (machine id: localgov_directories_channel_search_block) and **Directory facets** (machine id: facet_block:localgov_directories_facets) blocks should be made visible for the **Directory Channel** content type.  They can be added to a sidebar region (or equivalent) of the site theme.  Note that the facet_block:localgov_directories_facets block becomes available only after you have created at least one Directory entry content type.
+
+When using a theme other than the default LocalGov theme, the
```
Comment (Adnan-cds) [inline README.md:93]: Does this need to refer to the LocalGov Base theme?

Comment (finnlewis) [reply README.md:93]: I think this is OK, as the "deafult LocalGov theme" will now be localgov_base but may well become localgov_demo or some such....

**Review by finnlewis — APPROVED** [review]
Looks good to me!

File: README.md (hunk 1 above)
Comment (Adnan-cds) [reply README.md:93]: Thanks.

**Review by Adnan-cds — APPROVED** [review]
Looks okay to me.

**Pull request merged**